Core/clim-basic/pattern.lisp

```diff
@@ -388,7 +388,7 @@ Returns a pattern representing this file."
          (transformation (transformed-design-transformation effective-pattern))
          (inv-tr (invert-transformation transformation)))
     (multiple-value-bind (x y) (transform-position inv-tr x y)
-      (design-ink source-pattern (round x) (round y)))))
+      (design-ink source-pattern (floor (+ x 0.5)) (floor (+ y 0.5))))))
```

Comment:
Maybe add a short comment here (despite the commit messages containing the detailed explanation).

Reply:
Committed an update.

Reply:
This is too lenghy comment for such message. Let it something like this:
```
;; We don't use round because it doesn't round consistently and makes images
;; to have one extra pixel for coordinates having 0.5 fractions. -- loke 2018-01-05
```

also squash comment with the commit, they are one change. adding nick and date to comment is useful too.